reactioncommerce/reaction
# If default provider is not available, then add it.

Resolves #2018 

I decided on this method although I think it could be equally valid to say "you need to import the default method if you are going to modify them". Also maybe there is a better way to change that validation so you can still use the `ShippingMethod` schema?

### How to test:

1. Delete the included `Shipping.json`
1. `reaction reset && reaction`
1. Go to the Shipping dashboard
1. Enable Flat Rates
1. Observe there are no rates there
1. Click on the plus sign
1. Fill out the fields
1. Observe that you have added a rate.
